brain_games/scripts/brain_games.py

```diff
@@ -3,7 +3,6 @@
 
 
 def main() -> None:
-    """Outputs user greeting"""
     print("Welcome to the Brain Games!")
     welcome_user()
```

Comment:
`welcome_user` now prints `Welcome to the Brain Games!`, this line can be removed